ReactFrontEnd/README.md

```diff
@@ -1,32 +1,4 @@
-# Front End Website for Wallstreetbets Analytics tool
-
-The purpose of this website is to provide a simple site that can be used to visualize the data generated by the full Wallstreetbets Analytics tool.
 
 # How to use this repo
 
-### Installation
-
-1. clone this repo to your local dev environment.
-2. run the command `npm install` to install all of the project's dependencies.
-3. done.
-
-### localhost web development server
-
-1. install this repo according to the instructions above.
-2. run the command `npm run dev` to start the webpack 5 development server
-3. the site is now available on your local machine at http://localhost:8080
-
-### run a build for deploying to a webserver
-
-1. install this repo according to the instructions above.
-2. run the command `npm run build` to compile the code.
-3. upload the files in the `~/dist` folder to your webserver.
-4. the site is now available on your webserver.
-
-# Anything else I should know?
-
-Yes! This repo is set up to use the most recent version of React, React v17. Additionally it uses webpack 5 and babel for its production build. This is literally bleeding edge front end technology here! 
-
-# but what about IE 11?
-
-first, shut up. We hate IE 11 and if you use it you should upgrade. But with that said, yes, we support IE 11 but only so much as babel supports it. we don't support it with styles or more advanced visualization tools. Its not the 1800s anymore, you need to upgrade.
\ No newline at end of file
+don't. I saw that you are talking to angel investors and I have no intention of working for free for your bullshit startup. If you take on investment and want my input, you will have to pay my normal hourly rates which you probably can't afford.
```

Comment:
Full disclosure - I don't know what happened. Just sharing information posted on discord.

> Auzy 2/19 at 12:25 AM EST
@everyone After our conversation with zjz and seeing his response on reddit, we believe there may have been some miscommunication. We ARE NOT planning on selling out to some "angel investor" or anything. There has been one conversation that has occurred between us and said investor, and we are currently unsure what he would want out of it. It was merely intended to be mentioned in passing to illustrate the potential this idea has gathered within hours of its conception.
Anything that goes against our vision statement, goals, or ideals of leveling the playing field will NOT be tolerated. To address the "it's happening whether you help or not", that was first brought up by zjz (in different terms) and we were all in agreement. Someone, somewhere, is going to do this regardless of WSB's support. Do we want it to be us, who we know have good intentions, or the other people?

Reply:
I felt like I should respond directly, but Matt has already posted what I was going to say. I do apologize for any confusion there, but I believe there was some miscommunication. We are not planning to sell out to an investor or anything nor get free work for a "bullshit startup".

Reply:
no thanks.